addon/index.js

```diff
@@ -33,15 +69,22 @@ let RouterScrollMixin = Mixin.create({
   destroy() {
     reset();
 
+    if (requestId) {
+      window.cancelAnimationFrame(requestId);
+    }
+
     this._super(...arguments);
   },
 
   /**
    * Updates the scroll position
    * @param {transition|transition[]} transition If before Ember 3.6, this will be an array of transitions, otherwise
    * it will be a single transition
+   * @method updateScrollPosition
+   * @param {Object} transition
+   * @param {Boolean} recursiveCheck - if check until document height is same or lger than than  y
```

Comment:
Description of this param could read a little better.  `if "true", check until document height is >= y`.  Maybe replace `y` with what it actually is.